src/Response/SapiStreamEmitter.php

```diff
@@ -72,10 +76,17 @@ private function emitBody(ResponseInterface $response, $maxBufferLength)
      */
     private function emitBodyRange(array $range, ResponseInterface $response, $maxBufferLength)
     {
-        list($unit, $first, $last, $lenght) = $range;
+        list($unit, $first, $last, $length) = $range;
```

Comment:
No worries; I merged in order, and since they were the exact same change, it presented no conflict during merge. :smile: